scripts/e2e

```diff
@@ -10,4 +10,17 @@ if [ $# -eq 1 ]; then
   SPEC_ARG="--spec ${1}"
 fi
 
-CYPRESS_coverage=true CYPRESS_API_URL='http://139.59.134.103:1234' cy2 run ${SPEC_ARG} --group "$GREP_TAGS" --browser chrome --record --key rancher-dashboard --parallel --ci-build-id "$ID"
\ No newline at end of file
+# Define the command to use (cy2 or cypress)
+if [ "${TEST_DISABLE_DASHBOARD}" = "true" ] || [ "${TEST_DISABLE_DASHBOARD_LABEL}" = "true" ]; then
```

Comment:
Feels like we only need TEST_DISABLE_DASHBOARD, which would be populated by gh action or local dev

Reply:
As mentioned in the comment, it miss a bit of documentation, both code and Docusaurus one. Good to see first feedbacks as well 😄 

`TEST_DISABLE_DASHBOARD` is intended to be set in the GH settings in case we want to disable them for a long time.
`TEST_DISABLE_DASHBOARD_LABEL` is common mortals which have to skip it for quick fixes in case of issues.